minikubek8s.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Install kubectl
+echo "----------------------------kubectl checking----------------------------"
+export kubectl=$(which kubectl)
+if [ -f "$kubectl" ]; then
+    echo "$kubectl already exist...................................."
+else
+    echo "kubectl does not exist...................................."
+    echo "Installing kubectl........................................"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    kubectl version
+fi
+
+# Install Brew
+echo "-----------------------------brew checking------------------------------"
+export brew=$(which brew)
+if [ -f "$brew" ]; then
+    echo "$brew already exist......................................."
+else
+    echo "brew does not exist......................................."
+    echo "Installing brew..........................................."
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    brew cask install virtualbox
+fi
+
+# Install minikube
+echo "----------------------------minikube checking---------------------------"
+export minikube=$(which minikube)
+if [ -f "$minikube" ]; then
+    echo "$minikube already exist..................................."
+else
+    echo "minikube does not exist..................................."
+    echo "Installing minikube......................................."
+    curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 && chmod +x minikube
+    sudo mv minikube /usr/local/bin
+    minikube start --vm-driver=virtualbox
+    minikube status
+fi
+
+# Create yml config for Service account tiller
+cat <<EOF >./service-account.yml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: kube-system
+EOF
+
+# Create Dir for yml files
+echo "Creating Folder base for yml files........................"
+mkdir base
+mv ./service-account.yml base/
+
+# Apply service account
+echo "Apply Service-account....................................."
+kubectl apply -f base/service-account.yml
+kubectl get serviceaccounts -n kube-system
+
+# Create yml role-binding config for service account tiller 
+cat <<EOF >./role-binding.yml
```

Comment:
Sorry Samuel, what I meant in here is NOT create the file in this script, you can just have the file in the repo (you can delete lines from 63->76 and create a file in the repo).
`Devs` are going to have the entire repo pulled, you don't need to do the `cat <<EOF ...`

Reply:
Done